app/components/address-mapper.tsx

```diff
@@ -111,11 +120,216 @@ const AddressMapper: React.FC<AddressMapperProps> = ({
             tsunamiData={tsunamiData}
             liquefactionData={liquefactionData}
           />
-          <Box position="absolute" bottom={0}>
-            <Information />
-          </Box>
         </Box>
       </Box>
+      <Flex
+        w={{ base: "base", xl: "xl" }}
+        p={{
+          base: "24px 24px 24px 24px",
+          md: "36px 28px 16px 28px",
+          xl: "96px 128px 96px 128px",
+        }}
+        m="auto"
+        gap="46px"
+      >
+        <HStack alignItems={"start"}>
+          <div>
+            <Heading as="h2">
+              <Text
+                as="span"
+                textStyle="headerBig"
+                color="blue"
```

Comment:
The color of all headings slightly differs from the design. Should be #2B6CB0, whereas currently it is #2C5282

Reply:
This is updated as per Slack convo in follow-up PR #296.

Reply:
If that's ok with Micah, let's keep it as is.

Reply:
@mantuok FYI #296 addresses the heading colors